src/liballoc/collections/btree/node.rs

```diff
@@ -379,15 +379,21 @@ impl<BorrowType, K, V, Type> NodeRef<BorrowType, K, V, Type> {
         unsafe { &*(self.node.as_ptr() as *const NodeHeader<K, V>) }
     }
 
+    /// Returns whether this node is the root shared by all BTreeMap<K, V> instances
+    /// that don't contain any key-value pairs and never did.
```

Comment:
```suggestion
    /// that do not contain any key-value pairs and never did.
```

Reply:
I actually deleted that part because it is an abstraction at this level and nothing in the file decides over it.